packages/schema/src/interfaces.ts

```diff
@@ -183,6 +183,7 @@ export interface IJupyterGISModel extends DocumentRegistry.IModel {
   contentsManager: Contents.IManager | undefined;
   filePath: string;
 
+  getSettings(): any;
```

Comment:
Can we get better typing than this?

Reply:
Especially, the `any` is shadowing the fact that it can be `null`.

Reply:
Sure, i'll look into this